port/sockets.c

```diff
@@ -18,6 +20,7 @@
 #include <lwip/dhcp.h>
 #include <lwip/prot/dhcp.h>
 #undef ifreq
+#undef IFNAMSIZ
```

Comment:
this shouldn't probably be needed, but it depends on "untangling libphoenix includes" task, so I'm ok with merging this as-is.